lang/css/css.talon

```diff
@@ -23,12 +23,11 @@ rule <user.text>:
     name = user.formatted_text(text, "DASH_SEPARATED")
     insert("{name}: ")
 
-value <user.number_string> [{user.css_unit}]:
-    "{number_string}{css_unit or ''}"
+value <user.number_string> [{user.css_unit}]: "{number_string}{css_unit or ''}"
```

Comment:
I'm surprised this is being formatted as a single line, when [the very short `go usage` in `apps/visualstudio/visual_studio.talon` is being line wrapped](https://github.com/wenkokke/knausj_talon/commit/97913a8aa8219e02fb8f507a8986402c08e97316#diff-2f575b306ef5c73a031cfd7552f6c8637bcf25d8948f05cf36631f11a3802f45L75-R76). What's the logic here?

Reply:
The former fits onto an 80 column line, the latter is probably a bug relating to single-line commands found by themselves. I'll file the latter as a bug.

Reply:
Confusingly, my tests already check that `go usage` is formatted on a single line.